string[]  object_names
float64[] object_scales
sensor_msgs/PointCloud2[] object_point_clouds
geometry_msgs/Pose[] object_poses